### ISSUE TITLE
Create a Blog for detailed docs, tutorials, videos

### ISSUE THREAD

**Issue by soly2014** [InfyOmLabs/laravel-generator#118]
I want a video tutorial to deal with this package

**Comment by mitulgolakiya** [comment]
@soly2014 It's already in a plan where we are going to start a blog and will post videos and tutorial like how to use various features of a generator.

**Comment by mitulgolakiya** [comment]
Items need to be included:

- [ ] Tutorials on how to use various packages
- [ ] Video tutorials
- [ ] How package works Post
- [ ] How to extend Package
- [ ] How to develop your own packages as add-on to this package

**Comment by phillipmadsen** [comment]
I can start putting this together

**Comment by ztvMark** [comment]
The tutorials of this project are null, even in the blog there is none
---
I only  found this one
http://gauravghale.com.np/generator/

**Comment by mitulgolakiya** [comment]
This is a huge effort and needs significant time. Honestly, we are not generating any revenue from this and can only put a time which we get from our regular working hours or on weekends. This project has grown much and lots of issues or help questions are posted every day.

But apart from that, we try our best to get back asap. But response time is not guaranteed here. But still, we have started a blog here where we try to cover a few things.

https://blog.infyom.com/

Here we covered some datatable related things and updates. And we can not alone do this, this can be only possible via community help. so anyone is welcomed to write a blog post on stuff he/she finds interesting from issues here.
